client/app/components/dynamic-table/image-cell/index.js

```diff
@@ -0,0 +1,66 @@
+import { isUndefined } from 'underscore';
+
+const hasOwnProperty = Object.prototype.hasOwnProperty;
+
+function trim(str) {
+  return str.replace(/^\s+|\s+$/g, '');
+}
+
+function processTags(template, data, defaultColumn) {
+  return template.replace(/{{\s*([^\s]+)\s*}}/g, (match, column) => {
+    if (column === '@') {
+      column = defaultColumn;
+    }
+    if (hasOwnProperty.call(data, column) && !isUndefined(data[column])) {
+      return data[column];
+    }
+    return match;
+  });
+}
+
+function buildImgTag(column, row) {
+  const url = trim(processTags(column.imageUrlTemplate, row, column.name));
+  const width = parseInt(processTags(column.imageWidth, row, column.name), 10);
+  const height = parseInt(processTags(column.imageHeight, row, column.name), 10);
+  const title = trim(processTags(column.imageTitleTemplate, row, column.name));
+
+  const result = [];
+  if (url !== '') {
+    result.push('<img src="' + url + '"');
```

Comment:
Can someone inject script tags this way?

i.e., make `url` something like `"><script src="evil.js"><img src="`? Or it will be cleaned out by `$sanitize` below?

Reply:
I checked it - `$sanitize` removes all "bad" things, so it's safe.